jenkins/Pipeline.gy

```diff
@@ -1,7 +1,8 @@
 pipeline {
     agent { label 'pbench' }
     environment {
-        EXTRA_PODMAN_SWITCHES="-e COV_REPORT_XML=yes"
+        IMAGE_ROLE="ci"
```

Comment:
Do we really want to let `IMAGE_KIND` (and `IMAGE_REPO`) fall back to the default?

Reply:
For the CI job there are no other alternatives, so defaults are fine.

Reply:
I figured it was best not to duplicate anything since all of this IS for the CI jobs.  Since you asked, we now have them specified in two places.  =)

Reply:
One of the places is a "request"; the other is a "default" used in the absence of a request -- I think they are not duplicates because they aren't definitions.  (E.g., you could delete either one, and, barring changes elsewhere, things would continue to work.)

I think it's good to have defaults, because, when this stuff is used interactively (by humans), well-chosen defaults make life easier.  But, in the case of the CI, it doesn't need to depend on the defaults, and it costs very little to specify the values explicitly, so there's very little reason not to do so.

Before you went and made things so nicely general, there was only one choice, so we didn't need things like "requests" and "defaults".  But, now that we have options, I think the CI should ask for what it wants, and we can leave the defaults for people with fingers who have to type enough as it is.  :wink: